draft-ietf-quic-recovery.md

```diff
@@ -903,20 +915,22 @@ to OnPacketSent are described in detail above in {{sent-packets-fields}}.
 Pseudocode for OnPacketSent follows:
 
 ~~~
- OnPacketSent(packet_number, ack_eliciting, in_flight,
-              is_crypto_packet, sent_bytes):
-   largest_sent_packet = packet_number
-   sent_packets[packet_number].packet_number = packet_number
-   sent_packets[packet_number].time_sent = now
-   sent_packets[packet_number].ack_eliciting = ack_eliciting
-   sent_packets[packet_number].in_flight = in_flight
+ OnPacketSent(packet_number, pn_space, ack_eliciting,
```

Comment:
Using pn_space for the variable name and the type is confusing.  I think the enum should be kPacketNumberSpace

Reply:
Done.